lib/core/macros.nim

```diff
@@ -360,6 +360,13 @@ when defined(nimHasSignatureHashInMacro):
     ## the owning module of the symbol and others. The same identifier is
     ## used in the back-end to produce the mangled symbol name.
 
+when defined(nimHasSymBodyHashInMacro):
+  proc symBodyHash*(s: NimNode): string {.magic: "NSymBodyHash", noSideEffect.}
```

Comment:
Since this is now a VM callback there is no need to make this a ``magic``.

Reply:
sure and defined(nimHasSymBodyHashInMacro) is also no longer required.
All removed looks better now.